server/views/components/button-button/index.njk

```diff
@@ -0,0 +1,6 @@
+<button class="btn {{modifierClasses}}" {{attributes}}>
```

Comment:
should `attributes` not be a list of key => value pairs as you might want to put values in to?
i.e. `role="button"`?

We don't need it now - so happy to defer till we do.